app/main/page.tsx

```diff
@@ -36,19 +34,14 @@ const defaultWaveHandButtons = [
 ];
 
 const defaultAffirmations = [
-    'Say what I want to say, whatever happens will help me grow',
-    'I can take up space',
-    'I have an important voice',
-    'Feel the tension and proceed',
-    'I have the right to stutter',
-]
+    {id: 0, text: 'Say what I want to say, whatever happens will help me grow' },
```

Comment:
Do we already need ids even when this isn't connected to the DB?

Reply:
It seems to be the easiest way to find a replace the value of one item in the list. 

Set does not seem to support single item update.. 

See some examples for array manipulation [here](https://react.dev/learn/updating-arrays-in-state).